### PR TITLE
Fix #3378: Arrange the list of activities in alphabetical order in ActivityComponent.

### LINKED ISSUE #3378: Arrange the list of activities in alphabetical order in ActivityComponent
**Describe the bug**
The list of activities are not arranged in alphabetical order.

**To Reproduce**
1. Go to `ActivityComponent.kt`
2. See that at several instances the list is not in alphabetical order.

**Expected behavior**
The list of activities should be arranged in alphabetical order.

**Additional context**
File Reference:
https://github.com/oppia/oppia-android/blob/45e0bcb34678954ff85892c0ce50c942ff69c8aa/app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt#L73-L122

### PR DESCRIPTION
**Pull request by ayush0402**
## Explanation

Fixes #3378 
Arranged the list of activities in ActivityComponent.kt alphabetically.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by ayush0402** [comment]
@rt4914 PTAL.

File: app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt [hunk 1]
```diff
@@ -72,22 +72,21 @@ interface ActivityComponent {
 
   fun inject(addProfileActivity: AddProfileActivity)
   fun inject(adminAuthActivity: AdminAuthActivity)
-  fun inject(administratorControlsActivity: AdministratorControlsActivity)
   fun inject(adminPinActivity: AdminPinActivity)
+  fun inject(administratorControlsActivity: AdministratorControlsActivity)
```
Comment (rt4914) [inline app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt]: This should come above `AdminPin`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Just one change suggested.

File: app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt (hunk 1 above)
Comment (ayush0402) [reply app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt]: @rt4914 Done.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**